src/main/java/se/sundsvall/chat/api/model/ConversationRequest.java

```diff
@@ -0,0 +1,102 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a conversation containing participants, messages, and metadata.")
+public class ConversationRequest {
+
+	@ArraySchema(schema = @Schema(implementation = Participant.class, description = "The list of participants in the conversation."))
+	private List<Participant> participants;
+
+	@Schema(description = "A list of channel, reference, and context identifiers associated with the conversation.",
+		example = "[\"channel1-ref1-context1\", \"channel2-ref2-context2\"]")
+	private String channelId;
```

Comment:
The description says "a list of channel", but the variable (name + data type) says "a single channel ID".